Source/control.cpp

```diff
@@ -6,8 +6,8 @@ void *pDurIcons;
 void *pChrButtons;
 BOOL drawhpflag; // idb
 BOOL dropGoldFlag;
-WORD panbtn[8];
-WORD chrbtn[4];
+int panbtn[8];
+int chrbtn[4];

```

Comment:
What is the meaning of values held by both these arrays?  Can't they be bool too?

Reply:
They are added to a value before being passed to a function, I'm not sure of there exact meaning but having them as bool would definitely complicate the logic so I went with the simple change for now.